### ISSUE TITLE
no pictures for the author

### ISSUE THREAD

**Issue by reload81** [popeen/Booksonic-Air#26]
Thank you for giving us this great program. I'm using it now for about 2 weeks and have already added many audio books. I use it on the Univention Corporate server, based on Debian. It runs in a docker.

## Problem description

Unfortunately I can't see the pictures of the authors. I have saved the pictures in the corresponding folder of the author.

Am I doing something wrong?

## System information

2009.1.0 – September 15, 2020, based on Airsonic 8b28366
 * **Java version**: Version 8
 * **Proxy server**: None
 * **Client**: Booksonic Android-App; Firefox 82.0.3 (64-Bit)
 * **Language**: english

**Comment by vingenuity** [comment]
I would like to second this issue.

I'm running using the linuxserver.io Docker image, and can't get author images to display.

I've tried cover.jpg, cover.png, folder.jpg, and folder.png with no success.

**Comment by cypromis** [comment]
Any news on this one ?

**Comment by ehula** [comment]
Problem still exists.

**Comment by Ch4m311eom** [comment]
Same problem here.

**Comment by r0bis** [comment]
I agree, this would be a very nice improvement if the author pictures worked as well as the book pictures. Author `desc.txt` file works. It would be great if `cover.jpg` or `folder.jpg` type images could be used for authors.

**Comment by r0bis** [comment]
Where I should modify the code to allow the artist/author image to show. Something is done to generate `http://192.168.88.18:4040/coverArt.view?size=300&id=278` which points to the generated `coverArt.png`. If images are generated for the albums/books, it should be possible to do for the artist/author level as well.
